modules/ROOT/pages/faq.adoc

```diff
@@ -233,3 +233,9 @@ systemd:
     - name: docker.service
       mask: true
 ----
+
+== Are Fedora CoreOS disk images hybrid BIOS+UEFI bootable?
+
+The bare metal images we provide can be used for either BIOS (legacy) boot or UEFI boot, You can boot it in either BIOS (legacy) or UEFI mode regardless of what mode the OS will use once installed.
```

Comment:
I think you mixed up a bunch of statements here, the second part is usually found in the context of booting the live-ISO (in order to image a bare-metal disk).